src/core/a-animation.js

```diff
@@ -102,6 +102,7 @@ module.exports.AAnimation = registerElement('a-animation', {
 
     detachedCallback: {
       value: function () {
+        this.removeEventListeners(this.evt);
```

Comment:
I think we should make ```this.removeEventListeners``` to use this.evt internally instead of being passed as a parameter. @ngokevin what do you think?